020-Connect/020-JavaScript/020-drizzle.mdx

```diff
@@ -59,9 +99,10 @@ const result = await db
 
 The code imports required modules from the `drizzle-orm` and `@xata.io/drizzle` libraries. It also brings in the `eq` function from `drizzle-orm` for value comparison and the `getXataClient` function from a local xata module. The `getXataClient` function is used to acquire an instance of the Xata client.
 
-Currently, we recommend manually defining the schema like in the example above. We are working on utility functions to automatically generate the Drizzle model from the Xata database schema.
+Currently, we recommend manually defining the schema like in the example above. Alternatively, you can try to use `drizzle-kit` [experimental feature](https://orm.drizzle.team/kit-docs/commands#introspect--pull) to generate the schema from the database.
 
 ### Limitations
 
-- Drizzle migrations are not supported yet
-- Transactions are not supported yet
+- Transactions are not supported in the HTTP client.
```

Comment:
If there are open issues/tickets for these should we link to them?

Reply:
Transactions won't work unless we add something like the `batch` API that [neon introduced themselves](https://github.com/drizzle-team/drizzle-orm/pull/1895). I think it's fine as a known limitation for now, that has been the case always for drizzle http drivers until very recently that neon has added experimental support (the rest of drivers share the limitation still).